synthtool/gcp/common.py

```diff
@@ -369,7 +371,7 @@ def render(self, template_name: str, **kwargs) -> Path:
         _tracked_paths.add(template)
         return template
 
-    def _load_generic_metadata(self, metadata: Dict):
+    def _load_generic_metadata(self, metadata: Dict, relative_dir=''):
```

Comment:
I'm not sure how the `fs` API works for Python, but if this is a directory path, it would make more sense to me to be `.`, or `./` -- if these values work.